tests/metrics/network/dockerfile_image/Dockerfile

```diff
@@ -0,0 +1,46 @@
+#  This file is part of cc-oci-runtime.
+#
+#  Copyright (C) 2017 Intel Corporation
+#
+#  This program is free software; you can redistribute it and/or
+#  modify it under the terms of the GNU General Public License
+#  as published by the Free Software Foundation; either version 2
+#  of the License, or (at your option) any later version.
+#
+#  This program is distributed in the hope that it will be useful,
+#  but WITHOUT ANY WARRANTY; without even the implied warranty of
+#  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+#  GNU General Public License for more details.
+#
+#  You should have received a copy of the GNU General Public License
+#  along with this program; if not, write to the Free Software
+#  Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
+#
+
+# XXX: named version
+FROM ubuntu
+
+# Ensure packages are current, then install:
+#
+# - a basic development environment
+# - runtime build dependencies and additional general tooling
+#
+RUN apt-get -y update && \
+    apt-get install -y build-essential && \
+    apt-get install -y curl
+
+# Install smem (Memory reporting tool)
+RUN apt-get install -y smem
```

Comment:
I haven't looked out other Dockerfiles we maintain, and this is definitely a nitpick, but from reading Dockerfile best practices, they suggest doing all the apt-get's in one line, so for this it'd be:

RUN apt-get update && apt-get install -y \
  build-essential \
  curl \
  iperf3 \
  iperf \
  smem

At the very least, I'm not sure you need the -y for the initial apt-get update.